crates/pathfinder/src/rpc/types.rs

```diff
@@ -477,13 +479,15 @@ pub mod reply {
     }
 
     /// L2 transaction as returned by the RPC API.
+    #[skip_serializing_none]
```

Comment:
Did not know this works in containers, was expecting it to only work on fields. Feels a bit weird seeing it come before `#[derive(serde::Serialize)]` since I believe it's the one processing this?